appveyor is failing

The appveyor windows build is failing on master https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/37382340

Failing tests can be ignored for now. Resetting them won't help.

The issue slipped in because appveyor was accidentally disabled in the gui repo (now enabled).


```
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual struct QMetaObject const * __cdecl TransactionOverviewWidget::metaObject(void)const " (?metaObject@TransactionOverviewWidget@@UEBAPEBUQMetaObject@@XZ) [C:\projects\bitcoin\build_msvc\test_bitcoin-qt\test_bitcoin-qt.vcxproj]
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual void * __cdecl TransactionOverviewWidget::qt_metacast(char const *)" (?qt_metacast@TransactionOverviewWidget@@UEAAPEAXPEBD@Z) [C:\projects\bitcoin\build_msvc\test_bitcoin-qt\test_bitcoin-qt.vcxproj]
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual int __cdecl TransactionOverviewWidget::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@TransactionOverviewWidget@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) [C:\projects\bitcoin\build_msvc\test_bitcoin-qt\test_bitcoin-qt.vcxproj]
C:\projects\bitcoin\build_msvc\x64\Release\test_bitcoin-qt.exe : fatal error LNK1120: 3 unresolved externals [C:\projects\bitcoin\build_msvc\test_bitcoin-qt\test_bitcoin-qt.vcxproj]
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual struct QMetaObject const * __cdecl TransactionOverviewWidget::metaObject(void)const " (?metaObject@TransactionOverviewWidget@@UEBAPEBUQMetaObject@@XZ) [C:\projects\bitcoin\build_msvc\bitcoin-qt\bitcoin-qt.vcxproj]
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual void * __cdecl TransactionOverviewWidget::qt_metacast(char const *)" (?qt_metacast@TransactionOverviewWidget@@UEAAPEAXPEBD@Z) [C:\projects\bitcoin\build_msvc\bitcoin-qt\bitcoin-qt.vcxproj]
libbitcoin_qt.lib(overviewpage.obj) : error LNK2001: unresolved external symbol "public: virtual int __cdecl TransactionOverviewWidget::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@TransactionOverviewWidget@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) [C:\projects\bitcoin\build_msvc\bitcoin-qt\bitcoin-qt.vcxproj]
C:\projects\bitcoin\build_msvc\x64\Release\bitcoin-qt.exe : fatal error LNK1120: 3 unresolved externals [C:\projects\bitcoin\build_msvc\bitcoin-qt\bitcoin-qt.vcxproj]

--- a/build_msvc/libbitcoin_qt/libbitcoin_qt.vcxproj
+++ b/build_msvc/libbitcoin_qt/libbitcoin_qt.vcxproj
@@ -104,6 +104,7 @@
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactiondesc.cpp" />
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactiondescdialog.cpp" />
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactionfilterproxy.cpp" />
+    <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactionoverviewwidget.cpp" />
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactionrecord.cpp" />
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactiontablemodel.cpp" />
     <ClCompile Include="$(GeneratedFilesOutDir)\moc\moc_transactionview.cpp" />

